vertical-pod-autoscaler/pkg/apis/autoscaling.k8s.io/v1/types.go

```diff
@@ -107,8 +108,52 @@ type VerticalPodAutoscalerSpec struct {
 	// recommendation) or contain exactly one recommender.
 	// +optional
 	Recommenders []*VerticalPodAutoscalerRecommenderSelector `json:"recommenders,omitempty" protobuf:"bytes,4,opt,name=recommenders"`
+
+	// StartupBoost specifies the startup boost policy for the pod.
+	// +optional
+	StartupBoost *StartupBoost `json:"startupBoost,omitempty" protobuf:"bytes,5,opt,name=startupBoost"`
+}
+
+// StartupBoost defines the startup boost policy.
+type StartupBoost struct {
+	// CPU specifies the CPU startup boost policy.
+	// +optional
+	CPU CPUStartupBoost `json:"cpu,omitempty" protobuf:"bytes,1,opt,name=cpu"`
```

Comment:
Optional fields should be pointers:
kubernetes/community@master/contributors/devel/sig-architecture/api-conventions.md#optional-vs-required